src/esp/bindings_js/modules/navigate.js

```diff
@@ -167,10 +167,9 @@ class NavigateTask {
   }
 
   bindKeys() {
-    document.addEventListener("keyup", event => {
-      const key = String.fromCharCode(event.which).toLowerCase();
+    document.addEventListener("keydown", event => {
       for (let a of this.actions) {
-        if (key === a.key) {
+        if (event.key === a.key) {
```

Comment:
Some browsers still don't support event.key. Depends on what browsers we are targeting.

Ideally it should be

```
if (e.key) {
     const key = e.key;
} else {
    const code = e.which || e.keyCode;
    const key  = String.fromCharCode(code);
}
```

Reply:
It looks like it is supported by all the major browsers. We are limited to WebGL2 and WebAssembly so require a modern version.

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key#Browser_compatibility

Reply:
Ah, I forgot about that. Go ahead with the change then.